### PR TITLE
Added versioning steps to README

### PR DESCRIPTION
**Pull request by nunix**
Added the steps needed to create a new version, or in this case, update the `Stable` version with the content from the `Next` version.

This update is only triggered when a new version of Elemental is released, so the docs are up-to-date.

Signed-off-by: Nuno do Carmo <nuno.carmo@suse.com>

### CODE REVIEW

**Review by kkaempf — APPROVED** [review]
:+1:

File: README.md [hunk 1]
```diff
@@ -31,3 +35,29 @@ GIT_USER=<Your GitHub username> USE_SSH=true yarn deploy
 ```
 
 If you are using GitHub pages for hosting, this command is a convenient way to build the website and push to the `gh-pages` branch.
+
+## Docs versioning
+
+The versioning is done in two steps:
+
+1. Delete the entry `Stable` from the file `/versions.json`
```
Comment (fgiudici) [inline README.md:43]: do we still need this step if we call the release with the capitalized 'S' ?

**Review by fgiudici — APPROVED** [review]
great!

File: README.md (hunk 1 above)
Comment (nunix) [reply README.md:43]: yes, this is the only step needed. The capitalized `S` was in `docusaurus.config.js` file. So here it's already the new "way"

**Pull request merged**